tests/runner.py

```diff
@@ -37,10 +37,10 @@ def print_header(testname, arch):
     print("time        :  %s\n" % strftime("%c"))
 
 
-def run_single_test(testname, verbose, platform='stm32p103'):
+def run_single_test(testname, verbose, platform='stm32p103', debug=False):
     # platform = os.getenv('PLATFORM', 'qemu')
     cmd = ["make", "TEST=%s" % testname, "TARGET=%s" % platform,
-           "--file", "tests/Makefile", "clean_test", "all", "run"]
+           "--file", "tests/Makefile", "clean_test", "all", "dbg" if debug else "run"]
```

Comment:
How about changing from 'dbg' to 'debug' for better expression?

Reply:
I should not do this, but using argparse `args` to set up.

Reply:
I'll fix this in different PR.